File: 01_Leetcode_FAANG/Pratyush_Padihary/NumberOfGoodPairs.java
class Solution {
    public int numIdenticalPairs(int[] nums) {
        int c=0;
        int i,j;
     for(i=0;i<nums.length;i++)
     {
         for(j=0;j<nums.length;j++)
         {
             if(i<j)
             {
           if(nums[i]==nums[j])
               c++;
             }
         }
     }
        return c;
    }
}

File: 01_Leetcode_FAANG/CHANDAN_8144/Number_of_Good_Pairs.cpp
class Solution {
public:
    int numIdenticalPairs(vector<int>& nums) {
        int n=nums.size(),ctr=0;
        for(int i=0;i<n;i++){
            for(int j=i+1;j<n;j++){
                if(nums[i]==nums[j])
                    ctr++;
            }
        }
        return ctr;
    }
};
